build-system/tasks/e2e/index.js

```diff
@@ -50,6 +51,13 @@ async function setUpTesting_() {
     await buildRuntime();
   }
 
+  await buildBentoE2E({
+    fortesting: true,
+    localDev: true,
+    minify: false,
```

Comment:
i believe we run e2e on both minified and unminified code. if we could only pick one, it should be minified. but ideally we continue to run on both (i've often seen a bug only exposed via minification(